ci/test-upgrade/create-cluster.sh

```diff
@@ -15,7 +15,7 @@ fi
 
 echo "--- creating cluster"
 ./from/opstrace create ${OPSTRACE_CLOUD_PROVIDER} ${OPSTRACE_CLUSTER_NAME} \
-    --instance-config ci/test-upgrade/initial-cluster-config.yaml \
+    --instance-config ${INSTANCE_CONFIG} \
```

Comment:
This file is left over from a previous change to the instance config schema. Since the OPSTRA_CLI_VERSION_FROM version as been updated we can remove it. Note to self, don't forget to remove it.